jsonld/projects/C01-verified-carbon-standard-project.json

```
@@ -0,0 +1,59 @@
+{
+  "@context": {
+    "schema": "http://schema.org/",
+    "regen": "http://regen.network/",
+    "qudt": "http://qudt.org/schema/qudt",
+    "unit": "http://qudt.org/2.1/vocab/unit",
+    "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  "@type": "regen:Project",
+  "schema:name": "",
+  "regen:vcsProjectId": {
+    "@type": "xsd:unsignedInt",
+    "@value": 0
+  },
+  "regen:vcsProjectPage": {
+    "@type": "schema:URL",
+    "@value": ""
+  },
+  "regen:projectDeveloper": {
+    "@type": "regen:OrganizationDisplay",
+    "schema:name": "",
+    "schema:description": "",
+    "regen:showOnProjectPage": true
+  },
+  "regen:vcsProjectType": "",
+  "regen:projectActivity": {
+   "schema:name": "",
+    "schema:url": {
+      "@type": "schema:URL",
+      "@value": ""
+    }
+  },
+  "regen:offsetGenerationMethod": "",
+  "regen:vcsMethodology": {
+    "schema:name": "",
+    "schema:url": {
+      "@type": "schema:URL",
+      "@value": ""
+    }
+  },
+  "regen:projectSize": {
+    "qudt:unit": {
+      "@type": "qudt:Unit",
+      "@value": "unit:HA"
+    },
+    "qudt:numericValue": {
+      "@type": "xsd:double",
+      "@value": 0
+    }
+  },
```

@S4mmyb We may need location here. It is in the Project Page [standard](https://github.com/regen-network/regen-registry-standards/blob/3a565abd0366acea081b2de4b5a9ba24a14e2cce/shacl/projects/project-page.ttl#L48). 

@blushi Do we need to include location in the standard, or can we just consume it in the front end if it is available, like we do now with `ProjectPageShape`?

```suggestion
  },
  "schema:location": {
      ...location fields
  },
```

---

for reference, this is what Wilmot looks like

```
  "http://schema.org/location":
  {
      "id": "locality.6875563885184200",
      "bbox":
      [
          152.281925823569,
          -30.4188967883064,
          152.53051845185,
          -30.1967091580822
      ],
      "text": "Hernani",
      "type": "Feature",
      "center":
      [
          152.4184,
          -30.2986
      ],
      "context":
      [
          {
              "id": "postcode.8076814917761920",
              "text": "2453"
          },
          {
              "id": "region.19582497656831280",
              "text": "New South Wales",
              "wikidata": "Q3224",
              "short_code": "AU-NSW"
          },
          {
              "id": "country.9665923154346070",
              "text": "Australia",
              "wikidata": "Q408",
              "short_code": "au"
          }
      ],
      "@context":
      {
          "type": "@type",
          "@vocab": "https://purl.org/geojson/vocab#",
          "coordinates":
          {
              "@container": "@list"
          }
      },
      "geometry":
      {
          "type": "Point",
          "coordinates":
          [
              152.4184,
              -30.2986
          ]
      },
      "relevance": 1,
      "place_name": "Hernani, New South Wales, Australia",
      "place_type":
      [
          "locality"
      ],
      "properties": {}
  },
```

---

maybe also `schema:boundaries` if we want to have geojson maps

---

see my comment on slack https://regen-network.slack.com/archives/C01LX9E8QN8/p1647934515526819?thread_ts=1647612024.927119&cid=C01LX9E8QN8, we could use either `schema:location` or `schema:boundaries` for the map component

---

so just wanting to confirm that the todo item here is we need to add the geojson `schema:location` field, right? or is there something more to it?

---

yes that's all we need to add

---

done! i'm going to add new comments on these additions, i think it probably needs some feedback.